core/plugins/org.polarsys.capella.core.data.core.properties/src/org/polarsys/capella/core/data/core/properties/fields/TitleBlockBasicElementGroup.java

```diff
@@ -0,0 +1,175 @@
+      contentTextField = createTextField(textGroup, Messages.getString("NamedElement.ContentLabel"));
+    }
+  }
+
+  /**
+   * @param textGroup
+   * @param textLabel
+   */
+  private Text createTextField(Group textGroup, String textLabel) {
+    widgetFactory.createCLabel(textGroup, textLabel);
+
+    Text textField = widgetFactory.createText(textGroup, ICommonConstants.EMPTY_STRING);
+    textField.addFocusListener(this);
+    textField.addKeyListener(this);
+    textField.setLayoutData(new GridData(GridData.FILL_HORIZONTAL));
+
+    return textField;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public void loadData(EObject semanticElement, String name, String content) {
+    super.loadData(semanticElement, null);
+
+    if (null != semanticElement) {
+      if (null != nameTextField)
+        setTextValue(nameTextField, name);
+      if (null != contentTextField)
+        setTextValue(contentTextField, content);
+    }
+  }
+
+  protected void setTextValue(Text text, String value) {
+    text.setText(value);
+  }
+
+  /**
+   * @param textField
+   *          text field to be filled
+   */
+  @Override
+  protected void fillTextField(Text textField) {
+    if (textField.equals(nameTextField)) {
+      setNameValue(semanticElement, nameTextField.getText());
+    } else if (textField.equals(contentTextField)) {
+      setContentValue(semanticElement, contentTextField.getText());
+    }
+  }
+
+  private void setNameValue(EObject object, final Object value) {
+    AbstractReadWriteCommand command = new AbstractReadWriteCommand() {
+      @Override
+      public void run() {
+        ((DAnnotation) object).getDetails().put("Name:", value.toString());
+      }
+    };
+    executeCommand(command);
+  }
+
+  private void setContentValue(EObject object, final Object value) {
+    AbstractReadWriteCommand command = new AbstractReadWriteCommand() {
+      @Override
+      public void run() {
+        ((DAnnotation) object).getDetails().put("Content:", value.toString());
+      }
+    };
+    executeCommand(command);
+  }
+
+  /**
+   *
+   */
+  public void clearNameField() {
+    if (null != nameTextField) {
+      setDataValue(semanticElement, ModellingcorePackage.eINSTANCE.getAbstractNamedElement_Name(), "");
+      nameTextField.setText("");
+    }
+  }
+
+  /**
+   * @param enabled
+   *          whether or not the name text field is enabled
+   */
+  public void enableNameField(boolean enabled) {
+    if (null != nameTextField && !nameTextField.isDisposed()) {
+      nameTextField.setEnabled(enabled);
+    }
+  }
+
+  /**
+   * @param enabled
+   *          whether or not the summary text field is enabled
+   */
+  public void enableSummaryField(boolean enabled) {
+    if (null != contentTextField && !contentTextField.isDisposed()) {
+      contentTextField.setEnabled(enabled);
+    }
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public void setEnabled(boolean enabled) {
+    LockHelper.getInstance().enable(nameTextField, enabled);

```

Comment:
To test in collaborative mode to see if the cell is locked when being modifed by someone else.

Reply:
Will do.